### PR TITLE
libra/lip -> diem/dip

### PR DESCRIPTION
**Pull request by rexhoffman**
## Motivation

libra/lip -> diem/dip and a few other small cleanups

and now LIP-  -> DIP- as well

### Have you read the [Contributing Guidelines on pull requests]

Y

## Test Plan

CI

## Related PRs

None

### CODE REVIEW

File: config/management/operational/src/network_checker.rs [hunk 1]
```diff
@@ -92,15 +92,15 @@ pub mod tests {
         let op_tool = OperationalTool::new("unused-host".into(), ChainId::test());
 
         // Check invalid DNS
-        let addr = NetworkAddress::from_str("/dns4/libra/tcp/80").unwrap();
+        let addr = NetworkAddress::from_str("/dns4/diem/tcp/80").unwrap();
         op_tool.check_endpoint(addr).unwrap_err();
 
         // Check if endpoint responded with data
-        let addr = NetworkAddress::from_str("/dns4/libra.org/tcp/80").unwrap();
+        let addr = NetworkAddress::from_str("/dns4/diem.org/tcp/80").unwrap();
```
Comment (rexhoffman) [inline config/management/operational/src/network_checker.rs:99]: supposed to fail, diem.com succeeds so diem.org

File: language/stdlib/modules/doc/AccountFreezing.md [hunk 2]
```diff
@@ -619,6 +619,6 @@ Only (un)freeze functions can change the freezing bits of accounts [[H7]][PERMIS
 
 
 [//]: # ("File containing references which can be used from documentation")
-[ACCESS_CONTROL]: https://github.com/libra/lip/blob/master/lips/lip-2.md
-[ROLE]: https://github.com/libra/lip/blob/master/lips/lip-2.md#roles
-[PERMISSION]: https://github.com/libra/lip/blob/master/lips/lip-2.md#permissions
+[ACCESS_CONTROL]: https://github.com/diem/lip/blob/master/lips/lip-2.md
```
Comment (sausagee) [inline language/stdlib/modules/doc/AccountFreezing.md:622]: lib --> dip, https://github.com/diem/dip

**Review by sausagee — CHANGES_REQUESTED** [review]

File: language/stdlib/modules/doc/AccountFreezing.md (hunk 2 above)
Comment (rexhoffman) [reply language/stdlib/modules/doc/AccountFreezing.md:622]: heh, must have happened in the last few hours, sure.

File: specifications/crypto/README.md [hunk 3]
```diff
@@ -10,7 +10,7 @@ We do not specify any of one of these algorithms from first principles, but
 leverage and refer to existing specifications, while laying out exhastively
 where our implementation choices differ from the existant standards. Our
 implementation choices as described are expected to undergo future improvement
-through [Diem Improvement Processes](https://github.com/libra/lip) (LIPs).
+through [Diem Improvement Processes](https://github.com/diem/dip) (LIPs).
```
Comment (sausagee) [inline specifications/crypto/README.md:13]: DIPs

File: json-rpc/docs/client_checklist.md [hunk 4]
```diff
@@ -113,9 +113,9 @@ See [doc][5] for above concepts.
 - [ ] CLI connects to testnet for trying out features.
 
 [1]: https://github.com/diem/diem/blob/master/json-rpc/json-rpc-spec.md "Diem JSON-RPC SPEC"
-[2]: https://github.com/diem/lip/blob/master/lips/lip-5.md "LIP-5"
+[2]: https://github.com/diem/dip/blob/master/dips/dip-5.md "LIP-5"
```
Comment (sausagee) [inline json-rpc/docs/client_checklist.md:116]: LIP --> DIP

Comment (sausagee) [reply json-rpc/docs/client_checklist.md:116]: There are several more in this file.

File: specifications/crypto/README.md (hunk 3 above)
Comment (rexhoffman) [reply specifications/crypto/README.md:13]: Always one :D

Comment (rexhoffman) [reply specifications/crypto/README.md]: Did a find replace for lip -> dip as well, was trying to scope this to url, but only a few rs/md files talked about dips.

File: scripts/dev_setup.sh [hunk 5]
```diff
@@ -15,7 +15,7 @@ SHELLCHECK_VERSION=0.7.1
 HADOLINT_VERSION=1.17.4
 SCCACHE_VERSION=0.2.14-alpha.0
 #If installing sccache from a git repp set url@revision.
-SCCACHE_GIT='https://github.com/rexhoffman/sccache.git@19fef99c15765ea73460fd9ecb209c35313eac41'
+SCCACHE_GIT='https://github.com/rexhoffman/sccache.git@549babdd3866aa60dae01668c42ee00bf1e8c763'
```
Comment (rexhoffman) [inline scripts/dev_setup.sh:18]: Should have done this yesterday when I made the pr on sccache mergeable again -- and dev updated more deps today with more collisions.... 🤷

File: crypto/crypto/README.md [hunk 6]
```diff
@@ -32,4 +32,4 @@ Diem makes use of several cryptographic algorithms:
     └── unit_tests/         # Tests
 ```
 
-Note: This crate historically had support for BLS12381, ECVRF, and SlIP-0010, though were removed due to lack of use. The last git revision before there removal is 00301524.
+Note: This crate historically had support for BLS12381, ECVRF, and SDIP-0010, though were removed due to lack of use. The last git revision before there removal is 00301524.
```
Comment (rexhoffman) [inline crypto/crypto/README.md:35]: oops....

**Comment by rexhoffman** [comment]
@sausagee feel free to land right away, signing off for the evening.

**Review by sausagee — APPROVED** [review]
/land

**Comment by rexhoffman** [comment]
/land cluster test is pretty racy these days.

**Pull request closed without merging**